ApplicationPattern.yaml

```diff
@@ -589,6 +589,18 @@ paths:
                         'If provided, own TCP port, which has to be documented (but not be directly used for formulating the callback) into the TcpServer with
                         find [/core-model-1-4:control-construct/logical-termination-point=*-1-0-0-tcp-s-*/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-protocol] == https
                         update [/core-model-1-4:control-construct/logical-termination-point=*-1-0-0-tcp-s-*/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-port]'
+                    preceding-application-name:
+                      type: string
```

Comment:
shall we add the property "minLength: 3" for this attribute. Thanks.

Reply:
Changed as proposed (at both registry-office-application and preceding-application-name).